siga-ex/src/main/java/br/gov/jfrj/itextpdf/Documento.java

```diff
@@ -574,6 +569,66 @@ public static boolean getDocumento(OutputStream os, String uuid, ExMobil mob, Ex
 		}
 		return true;
 	}
+	
+
+	private static byte[] obterBytesDocumento(ExMobil mob, boolean estampar, ExArquivoNumerado an) 
+			throws DocumentException, IOException {
+		
+		String sigla = mob.getSigla();
+		if (an.getArquivo() instanceof ExMovimentacao) {
+			ExMovimentacao m = (ExMovimentacao) an.getArquivo();
+			if (m.getExTipoMovimentacao() == ExTipoDeMovimentacao.JUNTADA)
+				sigla = m.getExMobil().getSigla();
+		} else {
+			sigla = an.getMobil().getSigla();
+		}
+		
+		byte[] ab = !estampar ? an.getArquivo().getPdf() : Stamp.stamp(an
+				.getArquivo().getPdf(), sigla, an.getArquivo()
+				.isRascunho(), an.isCopia(), an.getArquivo().isCancelado(), an
+				.getArquivo().isSemEfeito(), an.getArquivo()
+				.isInternoProduzido(), an.getArquivo().getQRCode(), an
+				.getArquivo().getMensagem(), an.getPaginaInicial(),
+				an.getPaginaFinal(), an.getOmitirNumeracao(),
+				Prop.get("carimbo.texto.superior"), 
+				mob.getExDocumento().getOrgaoUsuario().getDescricao(), 
+				mob.getExDocumento().getMarcaDagua(), 
+				an.getMobil().getDoc().getIdsDeAssinantes());
+		
+		return ab;
+	}
+	
+	public static String obterTamanhoArquivosDocs(final DpPessoa dpPessoa, final DpLotacao dpLotacao,
+			final String arquivo, boolean completo, final boolean volumes, final boolean semmarcas)  throws Exception {
+		long bytes = 0;
+		boolean excedeuMB = Boolean.FALSE;
+		List<ExArquivoNumerado> ans = new ArrayList<ExArquivoNumerado>();
+		
+		if(Ex.getInstance().getConf().podePorConfiguracao(dpPessoa, dpLotacao, HABILITAR_PAGINACAO_PDF_DOC_COMPLETO)){
```

Comment:
Configuração é para pessoa/lotacao ou é para o ambiente.. Se for para o ambiente poderia ser uma property

Reply:
A configuração ficou agora por Prop (ambiente), sendo dois parâmetros:
		provider.addPrivateProperty("/siga.exibicao.paginada.pdf.completo.megabytes.tamanhomax", null);
		provider.addPrivateProperty("/siga.exibicao.paginada.pdf.completo.quantidade.documentos", null);